hole/css-colors-inverse.go

```diff
@@ -0,0 +1,27 @@
+package hole
+
+var cssFixedTests []test
+var cssCodesToNames = make(map[string][]string)
+
+func initCSS() {
```

Comment:
Just use `func init() {` and drop the `len(cssCodesToNames) < 1` check and then `initCSS()` calls.

As an aside, if we ever did need to lazily make a data structure at runtime we're better off using the std OnceFunc/Value which handles concurrent calls and stuff like that - https://pkg.go.dev/sync#OnceFunc